dart/test/keyword_test_keyword_test_generated.dart

```diff
@@ -0,0 +1,280 @@
+      KeywordsInUnionTypeId.fromValue(const fb.Uint8Reader().read(bc, offset));
+}
+
+class KeywordsInTable {
+  KeywordsInTable._(this._bc, this._bcOffset);
+  factory KeywordsInTable(List<int> bytes) {
+    final rootRef = fb.BufferContext.fromBytes(bytes);
+    return reader.read(rootRef, 0);
+  }
+
+  static const fb.Reader<KeywordsInTable> reader = _KeywordsInTableReader();
+
+  final fb.BufferContext _bc;
+  final int _bcOffset;
+
+  ABC get is_ => ABC.fromValue(const fb.Int32Reader().vTableGet(_bc, _bcOffset, 4, 0));
+  public get private => public.fromValue(const fb.Int32Reader().vTableGet(_bc, _bcOffset, 6, 0));
+  int get type => const fb.Int32Reader().vTableGet(_bc, _bcOffset, 8, 0);
+  bool get default_ => const fb.BoolReader().vTableGet(_bc, _bcOffset, 10, false);
+
+  @override
+  String toString() {
+    return 'KeywordsInTable{is_: $is_, private: $private, type: $type, default_: $default_}';
+  }
+
+  KeywordsInTableT unpack() => KeywordsInTableT(
+      is_: is_,
+      private: private,
+      type: type,
+      default_: default_);
+
+  static int pack(fb.Builder fbBuilder, KeywordsInTableT? object) {
+    if (object == null) return 0;
+    return object.pack(fbBuilder);
+  }
+}
+
+class KeywordsInTableT implements fb.Packable {
+  ABC is_;
+  public private;
+  int type;
+  bool default_;
+
+  KeywordsInTableT({
+      this.is_ = ABC.void_,
+      this.private = public.NONE,
+      this.type = 0,
+      this.default_ = false});
+
+  @override
+  int pack(fb.Builder fbBuilder) {
+    fbBuilder.startTable(4);
+    fbBuilder.addInt32(0, is_.value);
+    fbBuilder.addInt32(1, private.value);
+    fbBuilder.addInt32(2, type);
+    fbBuilder.addBool(3, default_);
+    return fbBuilder.endTable();
+  }
+
+  @override
+  String toString() {
+    return 'KeywordsInTableT{is_: $is_, private: $private, type: $type, default_: $default_}';
+  }
+}
+
+class _KeywordsInTableReader extends fb.TableReader<KeywordsInTable> {
+  const _KeywordsInTableReader();
+
+  @override
+  KeywordsInTable createObject(fb.BufferContext bc, int offset) => 
+    KeywordsInTable._(bc, offset);
+}
+
+class KeywordsInTableBuilder {
+  KeywordsInTableBuilder(this.fbBuilder);
+
+  final fb.Builder fbBuilder;
+
+  void begin() {
+    fbBuilder.startTable(4);
+  }
+
+  int addIs(ABC? is_) {
+    fbBuilder.addInt32(0, is_?.value);
+    return fbBuilder.offset;
+  }
+  int addPrivate(public? private) {
+    fbBuilder.addInt32(1, private?.value);
+    return fbBuilder.offset;
+  }
+  int addType(int? type) {
+    fbBuilder.addInt32(2, type);
+    return fbBuilder.offset;
+  }
+  int addDefault(bool? default_) {
+    fbBuilder.addBool(3, default_);
+    return fbBuilder.offset;
+  }
+
+  int finish() {
+    return fbBuilder.endTable();
+  }
+}
+
+class KeywordsInTableObjectBuilder extends fb.ObjectBuilder {
+  final ABC? _is_;
+  final public? _private;
+  final int? _type;
+  final bool? _default_;
+
+  KeywordsInTableObjectBuilder({
+    ABC? is_,
```

Comment:
This does not look very "darty" to me. Underscores at the end of variable names is a C++ thing, not a Dart thing.

My gut says this should be something like `$is`, since `$` is more commonly used by other generators in Dart like built_value and json_serializable.

Reply:
Done, I made keyword escaping work by prefixing `$`